MapboxAndroidDemo/build.gradle

```diff
@@ -12,8 +12,8 @@ android {
     applicationId "com.mapbox.mapboxandroiddemo"
     minSdkVersion 15
     targetSdkVersion 25
-    versionCode 38
-    versionName "5.0.0-beta.3"
+    versionCode 50
```

Comment:
This was not meant to match the versionName, this is a result of me playing around with bitrise google play publish workflow. FWIW current versionCode is 66.